r_basics_introduction/r_basics_introduction.md

```diff
@@ -670,8 +677,11 @@ Consider this line of code:
 ``` r
 covid_testing <- read_csv("covid_testing.csv")
 ```
-<div class = "warning">
+<div class = "care">
+<b style="color: rgb(var(--color-highlight));">A little encouragement...</b><br>
```

Comment:
```suggestion
<div class = "important">
<b style="color: rgb(var(--color-highlight));">Important note</b><br>
```

Agree that this is not a warning, but this doesn't feel even remotely like care to me... I think important note is probably our best bet?